src/styles/scenes/WalletListStyle.js

```diff
@@ -444,6 +444,23 @@ export const styles = {
     flex: 1,
     fontSize: 16,
     color: THEME.COLORS.GRAY_1
+  },
+  walletDetailsRowDifferenceNeutral: {
+    fontSize: 16,
```

Comment:
All of these font sizes need to have the scale function added. Also add scale to the dimensions in the parameters above from your previous pull request